build.gradle

```diff
@@ -93,27 +93,30 @@ subprojects {
-                errorprone_core      : "com.google.errorprone:error_prone_core:${errorProneVersion}",
-                errorprone_javac     : "com.google.errorprone:javac:${errorProneJavacVersion}",
-                grpc_api             : "io.grpc:grpc-api:${grpcVersion}",
-                grpc_context         : "io.grpc:grpc-context:${grpcVersion}",
-                grpc_protobuf        : "io.grpc:grpc-protobuf:${grpcVersion}",
-                grpc_stub            : "io.grpc:grpc-stub:${grpcVersion}",
-                guava                : "com.google.guava:guava:${guavaVersion}",
-                javax_annotations    : "javax.annotation:javax.annotation-api:1.3.2",
-                jmh_core             : "org.openjdk.jmh:jmh-core:${jmhVersion}",
-                jmh_bytecode         : "org.openjdk.jmh:jmh-generator-bytecode:${jmhVersion}",
-                jsr305               : "com.google.code.findbugs:jsr305:${findBugsJsr305Version}",
-                protobuf             : "com.google.protobuf:protobuf-java:${protobufVersion}",
-                protobuf_util        : "com.google.protobuf:protobuf-java-util:${protobufVersion}",
+                auto_value              : "com.google.auto.value:auto-value:${autoValueVersion}",
+                auto_value_annotation   : "com.google.auto.value:auto-value-annotations:${autoValueVersion}",
+                disruptor               : 'com.lmax:disruptor:3.4.2',
+                errorprone_annotation   : "com.google.errorprone:error_prone_annotations:${errorProneVersion}",
+                errorprone_core         : "com.google.errorprone:error_prone_core:${errorProneVersion}",
+                errorprone_javac        : "com.google.errorprone:javac:${errorProneJavacVersion}",
+                grpc_api                : "io.grpc:grpc-api:${grpcVersion}",
+                grpc_context            : "io.grpc:grpc-context:${grpcVersion}",
+                grpc_protobuf           : "io.grpc:grpc-protobuf:${grpcVersion}",
+                grpc_stub               : "io.grpc:grpc-stub:${grpcVersion}",
+                guava                   : "com.google.guava:guava:${guavaVersion}",
+                javax_annotations       : "javax.annotation:javax.annotation-api:1.3.2",
+                jmh_core                : "org.openjdk.jmh:jmh-core:${jmhVersion}",
+                jmh_bytecode            : "org.openjdk.jmh:jmh-generator-bytecode:${jmhVersion}",
+                jsr305                  : "com.google.code.findbugs:jsr305:${findBugsJsr305Version}",
+                prometheus_client       : "io.prometheus:simpleclient:${prometheusVersion}",
+                prometheus_client_common: "io.prometheus:simpleclient_common:${prometheusVersion}",
+                protobuf                : "com.google.protobuf:protobuf-java:${protobufVersion}",
+                protobuf_util           : "com.google.protobuf:protobuf-java-util:${protobufVersion}",
```

Comment:
Style change?

Reply:
Added prometheus entries, and a longer name and everything moved right